### PR TITLE
fix: enforce recipient limit in multipayment

### PR DESCRIPTION
**Pull request by alesmit**
## Summary

https://app.clickup.com/t/mxd0fb

## Checklist

-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] Tests
-   [ ] Ready to be merged

### CODE REVIEW

**Comment by faustbrian** [comment]
Task linked: [CU-mxd0fb [Multipayment] recipient limit not enforced](https://app.clickup.com/t/mxd0fb)

**Review by faustbrian — CHANGES_REQUESTED** [review]
Lacking some coverage

File: src/domains/transaction/components/AddRecipient/AddRecipient.tsx [hunk 1]
```diff
@@ -104,21 +102,21 @@ export const AddRecipient = ({
 	const { network, senderAddress, fee, recipientAddress, amount } = watch();
 	const { sendTransfer } = useValidation();
 
-	const senderWallet = useMemo(() => profile.wallets().findByAddress(senderAddress), [profile, senderAddress]);
-
 	const ticker = network?.ticker();
 	const exchangeTicker = profile.settings().get<string>(Contracts.ProfileSetting.ExchangeCurrency) as string;
 	const { convert } = useExchangeRate({ ticker, exchangeTicker });
 
+	const maxRecipients = wallet?.network().multiPaymentRecipients() ?? 0;
```
Comment (goga-m) [inline src/domains/transaction/components/AddRecipient/AddRecipient.tsx:109]: Hey @alesmit the Nullish coalescing operator might not be necessary as the `multiPaymentRecipients` will return 0 from sdk for default value :hmhm: https://github.com/ArkEcosystem/platform-sdk/blob/master/packages/platform-sdk/source/networks/network.ts#L278  

```suggestion
	const maxRecipients = wallet?.network().multiPaymentRecipients();
```

Comment (faustbrian) [reply src/domains/transaction/components/AddRecipient/AddRecipient.tsx:109]: I would assume this triggers when `wallet?` is true which means it'll run as `undefined ?? 0`

**Review by faustbrian — APPROVED** [review]

**Pull request merged**